app/backplane/sensor_module/sample.yaml

```diff
@@ -5,6 +5,7 @@ common:
   build_only: true
   integration_platforms:
     - sensor_module
+    - native_sim
 tests:
   app.default: {}
```

Comment:
pls rename to something like backplane.sensor_module.default. now that we're running multiple tests on one build server we run into name collisions if theyre all called app.default